.goreleaser.yml

```diff
@@ -32,7 +32,7 @@ builds:
 dockers:
   # Arch: amd64
   - image_templates:
-      - 'ghcr.io/google/osv-scanner:{{ .Tag }}-amd64'
+      - 'ghcr.io/{{ .Env.ACTOR }}/osv-scanner:{{ .Tag }}-amd64'
```

Comment:
Is this right? I'm not sure that the `github.actor` is going to be right if the repo is owned by an organization.

Reply:
I might use `github.repository_owner` instead, will it work?

Reply:
I renamed it as `OWNER` and used `github.repository_owner`.